contracts/utils/EnumerableSet4337.sol

```diff
@@ -3,6 +3,7 @@
 pragma solidity ^0.8.20;
 
 import "./AssociatedArrayLib.sol";
+import "forge-std/console2.sol";
```

Comment:
I think this new import was accidentally added and can be removed. Everything else in this PR looks good to me